From cca5b3510bfac9bc2a8967fc7a85b4b947cb1fe5 Mon Sep 17 00:00:00 2001
From: Kris Van Hees <kris.van.hees@oracle.com>
Date: Mon, 19 Nov 2018 17:18:23 +0000
Subject: [PATCH 10/20] dtrace: profile provider and test probe core components

Only Kconfig changes are needed here: everything else is purely modular.

Signed-off-by: Nick Alcock <nick.alcock@oracle.com>
Signed-off-by: Kris Van Hees <kris.van.hees@oracle.com>
Signed-off-by: Tomas Jedlicka <tomas.jedlicka@oracle.com>
Signed-off-by: Eugene Loh <eugene.loh@oracle.com>
Signed-off-by: David Mc Lean <david.mclean@oracle.com>
Signed-off-by: Vincent Lim <vincent.lim@oracle.com>
---
 kernel/dtrace/Kconfig | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/kernel/dtrace/Kconfig b/kernel/dtrace/Kconfig
index c1ec55d8750e..7b88206fe835 100644
--- a/kernel/dtrace/Kconfig
+++ b/kernel/dtrace/Kconfig
@@ -23,6 +23,12 @@ config DT_CORE
 
 if DT_CORE
 
+config DT_PROFILE
+	tristate "Profile Interrupt Tracing"
+	default m
+	help
+	  The profile and tick providers, firing probes at specific intervals.
+
 config DT_SDT
 	tristate "Statically Defined Tracing"
 	default m
@@ -54,6 +60,12 @@ config DT_DT_TEST
 	help
 	  A test provider used by the testsuite.
 
+config DT_DT_PERF
+	tristate "DTrace Performance Test Probe"
+	default m
+	help
+	  A test provider used for performance testing.
+
 config DT_DEBUG
 	bool "DTrace debugging"
 	default m
-- 
2.32.0

